backend/layers/processing/schema_migration.py

```diff
@@ -166,37 +169,37 @@ def collection_migrate(self, collection_id: str, collection_version_id: str, can
                 ).version_id.id
             else:
                 private_collection_version_id = collection_version_id
-
-            response = {
+            response_for_dataset_migrate = [
+                {
+                    "collection_id": collection_id,
+                    "collection_url": collection_url,
+                    "collection_version_id": private_collection_version_id,
+                    "dataset_id": dataset.dataset_id.id,
+                    "dataset_version_id": dataset.version_id.id,
+                    "execution_id": self.execution_id,
+                }
+                for dataset in datasets
+                if dataset.status.processing_status == DatasetProcessingStatus.SUCCESS
+                # Filter out datasets that are not successfully processed
+            ]
+            response_for_publish_and_cleanup = {
                 "can_publish": str(can_publish),
                 "collection_version_id": private_collection_version_id,
-                "collection_url": collection_url,
-                # ^^^ The top level fields are used for handling error cases in the AWS SFN.
```

Comment:
Now that we are using S3 this extra info does not need to be passed to `publish_and_cleanup`